### ISSUE TITLE
orderable config throws an expception at CollectionTreeRepository

### ISSUE THREAD

**Issue by aknEvrnky** [statamic/eloquent-driver#198]
Hi, I think I've found a bug that causes critic error. To reproduce it:

- create a new collection.
- set Orderable to true.
- try to reorder, create or any entry transaction on created collection.
- Call to undefined method Statamic\\Structures\\CollectionTree::toModel() at vendor/statamic/eloquent-driver/src/Structures/CollectionTreeRepository.php:25

my eloquent driver config:

| model      | driver |
| ----------- | ----------- |
| asset_containers      | eloquent       |
| assets   | eloquent        |
| blueprints   | file        |
| collections   | file        |
| collection_trees   | eloquent        |
| entries   | eloquent        |

rest of drivers are eloquent.

## my dependency & environment setup
```
php: 8.2
laravel: 10.20
statamic/cms: 4.19
statamic/eloquent-driver: 2.6.1
```

### the temporary solution
I've changed my collection_trees driver to file.

**Comment by ryanmitchell** [comment]
@aknEvrnky I've put a fix in [place here,](https://github.com/statamic/eloquent-driver/pull/199) do you want to check it out?

**Comment by ryanmitchell** [comment]
Should be fixed in 2.7.0
